docs/source/essentials/server.md

```diff
@@ -25,99 +17,43 @@ To install, run:
 
     npm install --save apollo-server graphql
 
-## Importing
+<h2 id="creating">Creating a server</h2>
 
-Depending on whether we are creating a new application or an existing application, the method of importing `apollo-server` will vary slightly since Apollo Server must adapt to the semantics of existing servers (e.g. Express, Hapi, etc.)
+The fastest way to get started with GraphQL is by creating a new server. Apollo Server will set an Express server up for you as long as you provide it with `typeDefs`, which is a string representing your GraphQL schema, and `resolvers`, which is a map of functions that implement your schema.
 
-Both import methods will use the `apollo-server` module we installed in the previous step, but existing applications will append the desired middleware as a path-based import (e.g. `apollo-server/<variant>`, where `<variant>` would be `express`, `koa`, etc.).
-
-In the below examples, we'll import two things from `apollo-server`:
+In the following examples, we'll import two things from `apollo-server`:
 
 * The `ApolloServer` class, which we'll use to instantiate and start the server.
 * The `gql` template literal tag, used for writing GraphQL within JavaScript code.
 
-These imports will be explained in the next sections.
-### New applications
-
-For new applications, it's not necessary to use a middleware variant and Apollo Server comes ready to run.  In this case, we'll **add this line** to the top of an application's main entry-point then skip ahead to "Import type definitions" below:
-
 ```js
 const { ApolloServer, gql } = require('apollo-server');
-```
-
-### Existing applications (Middleware)
```

Comment:
This middleware option is very important. By removing it, we leave developers who are adding Apollo Server to an existing application in the dark.  All users of Apollo Server 1.0 are using the middleware approach.

I understand streamlining this, but essentials should have a path for those users otherwise we have to duplicate all this related content elsewhere.  It seems like a single callout here for middleware users is the cleanest way to do this?